### DISCUSSION TITLE
Feature Request: Time Series panel: provide median as legend value

### DISCUSSION THREAD

**Discussion by wfjm** [grafana/grafana#43835] [Dashboards, Panels & Visualizations]
The legend of the time series panel provides legend values: `min`,`max`,`mean`, and many more.

We have data series with occasional outliers.
In this situation the `mean` value can quickly be distorted and essentially useless. 
As Wikipedia  puts it in _robust statistics_:
> The mean is not a robust measure of central tendency.
> The median is a robust measure of central tendency.

So I'd very much appreciate to also have the `median` available as legend value.
A combination of `min`,`max` and `median` is in case of outliers a very good way to capture the essentials.

**Comment by ahirner** [comment]
It seems relatively easy to implement and I'm interested as well.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
